guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>"<code>use_data</code> parameter can be specified to explicitly call the country or dataset to query. This parameter can be specified to provide an additional "performance hint" to the service. "</p><ul><li>In order to explicitly call the country or dataset to query, the <code>use_data</code>  parameter can be specified to ...</li></ul><p> </p><p>"Specifying a specific dataset or country through this parameter will potentially improve response time. "</p><ul><li>Narrowing down the query to a specific dataset or country through this parameter....</li></ul><p> </p><p>"Let's look at an example of indicating to the service that all input features in the study_areas parameter describe locations or areas in the U.S. <code>USA_ESRI_2019</code> dataset."</p><ul><li>Let's look at an example of how <code>use_data</code>  is used to indicate to the service that all input features in the <code>study_areas</code> parameter describe locations or areas only discoverable in the U.S. <code>USA_ESRI_2019</code> dataset.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>